deployment.go

```diff
@@ -10,8 +10,9 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-
+	"github.com/ubclaunchpad/inertia/common"
 	"github.com/ubclaunchpad/inertia/client"
+	"os"
```

Comment:
Are you using a Go linter with your IDE? For me at least, the Golang extension for [Visual Studio Code](https://code.visualstudio.com) will automatically sort these imports correctly and add/remove things as needed (the native `os` import should be above these third party imports)